openwisp_users/base/models.py

```diff
@@ -59,6 +60,11 @@ class AbstractUser(BaseUser):
     notes = models.TextField(
         _('notes'), help_text=_('notes for internal usage'), blank=True
     )
+    language = models.CharField(
+        max_length=10,
```

Comment:
Let's shorten this to 8.

7 seems the maximum used by the languages configured in Django:
https://github.com/django/django/blob/main/django/conf/global_settings.py#L56-L153

We can keep 1 spare character in case some crazy language comes out but there shouldn't be other languages which have codes longer than that.